File: 프로그래머스/0/120824. 짝수 홀수 개수/짝수 홀수 개수.js
function solution(num_list) {
    let answer = [0, 0];
    
    for (let i = 0; i < num_list.length; i+=1) {
        (num_list[i] % 2 === 0) ? answer[0] += 1 : answer[1] += 1;
    }
    
    return answer;
}

File: 프로그래머스/0/120824. 짝수 홀수 개수/짝수 홀수 개수.kt
class Solution {
    fun solution(num_list: IntArray): IntArray {
        var num_even = 0
        var num_odd = 0
        for(number in num_list) {
            if(number % 2 == 0) num_even += 1 else num_odd += 1
        }
        return intArrayOf(num_even, num_odd)
    }
}